component/card.css

```diff
@@ -0,0 +1,92 @@
+.card{
+    position: relative;
+    background-color: var(--secondary-color);
+    border-radius: 1rem;
+    margin: 1rem;
+    max-width: 25rem;
+    width: 100%;
+  }
+
+  
+  .image-responsive{
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    border-radius: 1rem 1rem 0 0;
+  }
+  
+  .card-btn-container{
+    display: flex;
+    flex-wrap: wrap;
+  }
+
+  .card-container{
+      margin: 0 0 1rem 1rem;
+  }  
+
+  .card-text-container{
+    /* margin: 1rem; */
+    padding: 1rem;
+   
+  }
+  .card-heading{
+    font-size: 1.3rem;
+    /* font-weight: 500; */
+    margin: 0;
+  }
+  .card-subheading{
+    font-size: 0.9rem;
+    font-weight: 700;
+    margin: 0 0 0.5rem 0;
+  }
+  
```

Comment:
Fix spacing here, and also remove comments if they are not needed

Reply:
Done the Changes